charts/charts/puris-frontend/templates/deployment.yaml

```diff
@@ -34,10 +34,24 @@ spec:
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           env:
-            - name: VITE_APP_NAME
-              value: "{{ .Values.app.viteAppName }}"
-            - name: VITE_BASE_URL
-              value: "{{ .Values.app.viteBaseUrl }}"
+            - name: APP_NAME
```

Comment:
please change prefix from app to "puris-frontend". prefixes should follow the same convention. e.g. frontend and backend or even better puris-frontend and puris-backend